include/swift/AST/DiagnosticsSema.def

```diff
@@ -1499,7 +1499,7 @@ ERROR(access_control_extension_more,none,
       "be declared %select{%error|fileprivate|internal|public|%error}2",
       (AccessLevel, DescriptiveDeclKind, AccessLevel))
 ERROR(access_control_extension_open,none,
-      "extensions cannot use 'open' as their default access; use 'public'",
+      "Extensions cannot use 'open' as their default access; remove the default access and mark methods as 'open' instead",
```

Comment:
I think the guideline is all diagnostics should start with lower case :)

Reply:
Thanks! I reverted it to lowercase :)